src/main/java/org/opensearch/neuralsearch/processor/normalization/MinMaxScoreNormalizationMethod.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.neuralsearch.processor.normalization;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+import org.apache.lucene.search.ScoreDoc;
+import org.apache.lucene.search.TopDocs;
+import org.opensearch.neuralsearch.search.CompoundTopDocs;
+
+import com.google.common.primitives.Floats;
+
+/**
+ * Abstracts normalization of scores based on min-max method
+ */
+public class MinMaxScoreNormalizationMethod implements ScoreNormalizationMethod {
+    static final float MIN_SCORE = 0.001f;
```

Comment:
Can we make it either public or private?

Reply:
ack